runtime/runtime-params-estimator/src/main.rs

```diff
@@ -28,6 +28,9 @@ struct CliArgs {
     /// to generate appropriate data.
     #[clap(long)]
     home: Option<PathBuf>,
+    /// If a home directory has been specified, set this flag if it needs to be initialized.
+    #[clap(long)]
+    init_home: bool,
```

Comment:
Can we just infer/DWIM this? Ie, if the directory is empty, it is populated

Reply:
Yes that seems like a better approach here!